src/components/views/rooms/RoomPreviewBar.tsx

```diff
@@ -57,6 +57,9 @@ enum MessageCase {
 }
 
 interface IProps {
+    // The id of the room to be previewed
```

Comment:
```suggestion
    // The id of the room to be previewed, if it is known.
    // (It may be unknown if we are waiting for an alias to be resolved.)
```